### ISSUE TITLE
Create and implement 5 more User Stories

### ISSUE THREAD

**Issue by Majjalpee** [MunchApp/munch-app#43]
Thinking we can do:

- As a food truck enthusiast, I want to be able to log into the app and have my details saved, so that I can be connected to the app and local food trucks in the area.

- As a food truck owner, I want to be able to log into the app and have my details saved, so that I can quickly register my owned food trucks and start using the apps functionality. 

- As a general user (user not logged in), I want to be able to see individual food truck profiles so that I can get a better understanding of if I want to eat there. 

-  As a general user, I want to be able to see a general mapping of all the food trucks around my location, so that I can tell what places are close to eat at. 

- As a general user, I want to be able to get the directions to a food truck of my choosing, so that I can go there and eat. 

TLDR: implement login functionality, food truck profile pages, populate map with food trucks, integration with google maps or open google maps activity to show directions to a food truck.
